test/unit/dropin-model.js

```diff
@@ -1591,5 +1591,72 @@ describe('DropinModel', () => {
         });
       }
     );
+
+    describe('hiddenVaultedPaymentMethodTypes options', function () {
+      test.each([
+        {
+          hiddenVaultedPaymentMethodTypes: [],
+          expected: ['AndroidPayCard', 'ApplePayCard', 'VenmoAccount']
+        },
+        {
+          hiddenVaultedPaymentMethodTypes: ['paypal'],
+          expected: [
+            'AndroidPayCard',
+            'ApplePayCard',
+            'PayPalAccount',
+            'VenmoAccount'
+          ]
+        },
+        {
+          hiddenVaultedPaymentMethodTypes: [
+            'unknown',
+            'googlePay',
+            'card'
+          ],
+          expected: [
+            'AndroidPayCard',
+            'ApplePayCard',
+            'CreditCard',
+            'VenmoAccount'
+          ]
+        },
+        {
+          // eslint-disable-next-line no-undefined
+          vaultedPaypalAccountsDisabled: undefined,
+          expected: ['AndroidPayCard', 'ApplePayCard', 'VenmoAccount']
+        }
+      ])(
+        'when hiddenVaultedPaymentMethodTypes is set to $vaultedPaypalAccountsDisabled',
```

Comment:
Can you refine the test name to include some manner of expectation the test will evaluate? That is, when `hiddenVaultedPaymentMethodTypes` is set to <something>, what do we expect?

Reply:
> Can you refine the test name to include some manner of expectation the test will evaluate? That is, when `hiddenVaultedPaymentMethodTypes` is set to <something>, what do we expect?

Gracias por todo

Reply:
@ale-grosselle if these last couple things get addressed then we can get this merged.

Reply:
Hey @cgdibble thanks for the review! 
In these days I'm going to implement all

Reply:
> Can you refine the test name to include some manner of expectation the test will evaluate? That is, when `hiddenVaultedPaymentMethodTypes` is set to _something_, what do we expect?

braintree.client.create({
  authorization: 'authorization'
}).then(function (clientInstance) {
  return braintree.paypalCheckout.create({
    client: clientInstance
  });
}).then(function (paypalCheckoutInstance) {
  return paypal.Button.render({
    env: 'production', // or 'sandbox'

    payment: function () {
      return paypalCheckoutInstance.createPayment({
        // your createPayment options here
      });
    },

    onAuthorize: function (data, actions) {
      // some logic here before tokenization happens below
      return paypalCheckoutInstance.tokenizePayment(data).then(function (payload) {
        // Submit payload.nonce to your server
      });
    }
  }, '#paypal-button');
}).catch(function (err) {
 console.error('Error!', err);
});

Reply:
@cgdibble done, could you check please?